internal/api/token.go

```diff
@@ -494,3 +516,28 @@ func (a *API) clearCookieToken(config *conf.GlobalConfiguration, name string, w
 		Domain:   config.Cookie.Domain,
 	})
 }
+
+func validateClaims(outputClaims map[string]interface{}) error {
```

Comment:
just to clarify my understanding, this function is used to check if the access token returned from the hook has the minimal set of required claims right?

Reply:
Yup that's right, open for discussion though. I'll also add a comment to make this clearer